pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Creates a new project from command line."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--code',
+            action='store',
+            dest="code",
+            help='Project code',
+        )
+        parser.add_argument(
+            "--name",
+            action="store",
+            dest="name",
+            help="Project name",
+        )
+        parser.add_argument(
+            "--quality",
+            action="store",
+            dest="quality",
+            default="standard",
+            help="Quality management",
+        )
+        parser.add_argument(
+            "--filetype",
+            action="store",
+            dest="filetype",
+            help="File type: [ po/pot | xliff | xlf | ts | lang ]",
+        ) 
+        parser.add_argument(
+            "--structure",
+            action="store",
+            dest="structure",
+            default="gnu",
+            help="Catalogue structure: [ automatic | gnu | no-gnu | filesystem ]",
+        )
+        parser.add_argument(
+            "--sourcelang",
+            action="store",
+            dest="sourcelang",
+            default="en",
+            help="Source language. Examples: [ en | es | fr | ... ]",
+        )
+        parser.add_argument(
+            "--contact",
+            action="store",
+            dest="contact",
+            help="Contact for errors in lines. Example: admin@mail.com",
+        )
+        parser.add_argument(
+            "--disabled",
+            action="store",
+            type=bool,
+            dest="disabled",
+            default=False,
+            help="Does the project start disabled? [ True | False ]",
+        )
+
+    def handle(self, **options):
+        self.create_project(**options)
+
+    def create_project(self, **options):
+        """Immitates the behaviour of the admin interface for creating a project from command line."""
+
+	from pootle_language.models import Language
+	from pootle_project.models import Project
+
+	source_language_obj = Language.objects.get(code=options["sourcelang"]) 
+	
+        if "name" not in options:
+          options["name"] = options["code"].capitalize()
+		
+	new_project = Project.objects.create(code=options["code"], fullname=options["name"], source_language=source_language_obj)
```

Comment:
you will need to make this statement multiline or it will fail pep8

also re fullname, rather than setting above as you have done you could do...

...`, fullname=options["name"] or options["code"].capitalize(), `...